test/unit.js

```diff
@@ -118,4 +118,23 @@ describe('SQL', () => {
       assert.deepStrictEqual(statement.values, [123])
     })
   })
+
+  describe('Auto Escape', () => {
+    it('should escape the table name', () => {
+      assert.strictEqual(SQL`SELECT * FROM '${`table_${123}`}'`.sql, 'SELECT * FROM `table_123`')
```

Comment:
MySQL uses backticks, but but Postgres and the ANSI SQL standard use double quotes for identifiers. So I don't think auto-escaping is possible like this.

Reply:
it is, please read my previous comment